### DISCUSSION TITLE
personal-development-tir-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#50] [Announcements]
# personal-development-tir-1403/

ماه تیر هم به آخر خودش رسید و طبق قراری که با خودم گذاشتم، گزارش توسعه فردی این ماهم رو منتشر می‌کنم. در کل ماه خوبی بود و ازش راضی بودم.
بعد سال‌ها هم حال و هوای محرم یزد رو تجربه کردم. یزد تو نیمه شعبان و محرم حال و هوای جالبی داره؛ خصوصاً که غذاهایی مثل «آش گندم» رو بیشتر تو همین ایام میشه پیدا کرد و ظاهراً تو شهرهای دیگه غذایی به این شکل ندارند (البته تو تهران یه سری غرفه اصطلاحاً یزدی داشتیم که می‌گفتند آش گندم می‌فروشند اما بعید می‌دونم هیچ وقت آش گندم خورده باشند - چه برسه به این که بپزند).

https://aprd.ir/personal-development-tir-1403/

**Comment by szareui** [comment]
امیر عزیزم 

یکی از مزایای دورکاری همینه که زمان بیشتری رو توی خونه با هم می‌گذرونیم و بیشتر از همدیگه یاد می‌گیریم و خوب ابعادی رو از هم میبینیم که قبلا نمی‌دیدیم.

اخیرا متوجه شدم که به خوبی حل مسئله میکنی و بلدی سوال درست بپرسی. مشکل اصلی خیلی از ماها همینه که برای حل مسئله به سرعت میریم سراغ راه حل، در حالی که حتی مسئله رو درست متوجه نشدیم و داریم مشکل اشتباهی رو حل می‌کنیم. 

چیزی که خیلی برام جالب و جذابه اینه که امیری که ۶ سال پیش می‌شناختم یه آدم فنی قوی اما تک بعدی بود، ولی امروز به قدری ابعاد مختلف شخصیتت رو پرورش دادی که تاثیرش هم توی زندگی شخصیت هم حرفه‌ایت مشهوده. چقدر خوشحالم که متمم خوانی چنان تاثیر عمیقی گذاشته که میتونی اینقدر قشنگ فکر کنی.

**Reply by pourmand1376** [reply]
ممنون صدیقه جان.

الان که بیشتر فکر می‌کنم فقط هم تأثیر متمم‌خوانی نبوده. بخش بزرگیش به خاطر صحبت‌هایی بوده که با هم داشتیم. یه ویژگی‌ای که تو داری و من خیلی کمتر دارم، پردازشی هست که روی داده‌هات انجام می‌دی؛ یعنی راجع به هر مسئله‌ای اگر کوچکترین چیزی یاد بگیری، کلی بهش فکر می‌کنی و نتایج جالبی هم ازش بدست میاری. این از مواردی هست که همیشه برام عجیب بوده. 

یعنی برخلاف تو، من حجم داده ورودی‌ای که دریافت می‌کردم و می‌کنم شاید خیلی بیشتر باشه. ولی تو خیلی پردازش‌های بهتری روش انجام می‌دی. 

اینه که به نظرم ذهن شفاف‌تری داری و با این که تو هر مسئله‌ای نظر نمیدی، جاهایی که نظر میدی، نظراتت پخته‌تر و عمیق‌تر هست.

**Comment by hiwasham** [comment]
مرسی امیر سرنخ های جالبی همیشه برمیدارم از گزارش توسعه هات. میذارمشون داخل Obsidian.

- نمیدونستم کتاب Unleash Your Complexity Genius ترجمه شده. منم اتفاقاً ماه پیش خوندمش و اونقدر برام جالب بود که رفتم کتابهای دیگه اش رو هم بررسی کردم. 
کتاب Simple Habits for Complex Times رو هم دوست داشتم.
البته برای من بخشی از مطالعات فلسفی و روانشناختیم میان پشت این مفاهیم و حس میکنم دید بهتری بهم میدن. حس میکنم اگه اونهارو ندونیم، درک و اجرای تکنیکهایی که میگه سخت تر میشه.
اما به هرحال کتابهای خیلی خوبی دارن مینویسن
https://www.cultivatingleadership.com/books
یه ایرانی هم توی لیست نویسنده هاشون هست

**Reply by pourmand1376** [reply]
ممنون هیوا جان.

کلا چند وقته دارم سعی می‌کنم که کتاب‌های مرتبط با مفهوم «پیچیدگی» رو تو برنامه‌ام بگذارم. هم این که به نظرم برای درک جهان پیچیده جواب‌های بهتری نسبت به بقیه مدل‌ها داره و هم این که شاید ترکیب دانش پیچیدگی و هوش‌ مصنوعی چیز جالبی ازش دربیاد. 

چند وقت پیش که رفته بودم کتابفروشی‌های انقلاب دو تا کتاب پیچیدگی هم گرفتم و متاسفانه آنقدر بد ترجمه شده‌اند که جفت‌شون رو باید بندازم دور. نمی‌دونم چی شد که اون شب کیفیت بد ترجمه رو نفهمیدم و بابت همچین چیزهای بد کیفیتی وقت، انرژی و هزینه گذاشتم. 

فقط یه سوال برام پیش اومده. شما کتاب‌هایی مثل Simple Habits for Complex times رو چطوری می‌خونی؟ چون پیدا کردن نسخه انگلیسی اصلی چاپی خیلی کتاب‌ها کار حضرت فیله. و خواندن تو لپ‌تاب و کیندل هم حداقل برای من جواب نیست. گفتم شاید سایتی چیزی برای خرید می‌شناسی که همیشه از اون استفاده می‌کنید.

**Reply by hiwasham** [reply]
راستش امیر جواب دادن به این سوال برام سخته. میتونم کلی توضیح بنویسم اما اگر بخوام مفید باشه نمیدونم چطوری توضیح باید بدم. 
بذار بگم جریان چیه
من شش ساله دارم از obsidian استفاده میکنم. البته این ابزارم هست. روش یه چیزی بین Zettelkasten و کتابهای تیاگو فورته و نرم افزار Supermemo و incremental reading و چند تا موضوع مشابه دیگه هست  که البته خودم شخصی شون کردم
الان شش ساله اینکارو میکنم. داخل ابسیدین حدود 60 هزار نوت دارم براش. که البته اینها راس های یه گراف هست. یالهاش خیلی مهمتره. هزاران نوت بین اینها ساختم به روشهای مختلف.
اگه ازم بپرس چطوری اینکارو میکنی، نمیتونم توضیح بدم
سالهای اول هم کلا توی دیوار بودم. بعدها میگفتم کاش یکی اینهارو آموزش میداد که انقدر آزمون خطا نکنم. دوره و کتاب هم زیاد داشتم براش اما در نهایت زحمت توسعه روشه برای شخص خودم با خودم بود.
در نهایت الان روشم اونقدر غیرعادیه که نمیدونم چطوری باید توضیحش بدم. در قالب یه دوره چند ده ساعته و مثالهای واقعی شاید بتونم.
به هرحال نکته مهم اینه که به روش زتلکستن هر چی که برام مهمه میره داخل یه سری نوت (انواع نوت داریم). روش CODE تیاگو فورته رو هم شخصی سازی کردم. برای مواردی که مهمتره و فرصت میشه incremental reading انجام میدم. حتماً هر چیزی رو به ریزترین جزئیات زندگی روزمره م وصل میکنم. بعضی نوتهارو ده ها بار در طول سال بازبینی میکنم و هربار کمی تغییرشون میدم (یعنی شخصی تر و کاربردی ترش میکنم برای خودم) و وصلش میکنم به نوت های دیگه مرتبط. مثلا یه بخش از کتاب بالا رو ممکنه وصل کنم به یه مبحث روانشناسی، یه کامنت محمدرضا، یکی از کتابهای نیچه، چند تا نقل قول پراکنده از آدمهای مختلف، یه سری تجربیات مرتبط خودم، فلان موضوع برنامه نویسی، ، فلان کلیپ طنز یوتیوب و ده ها نوت دیگه (بعضی نوتهام ممکنه بیشتر از 100 لینک ورودی یا خروجی بهشون وصل باشه)
در طی این پروسه هم حس میکنم ذهنم ورزیده تر شده. یعنی باسرعت بالا و اتوماتیک اینکارهارو میکنم. 
یه سری چیزهارو هم ممکنه روی برگه یا داخل ایپد یا جاهای دیگه بنویسم اما به روش GTD یا PARA و ... (که هر کدوم مبحثهای مفصلی هستن) اینهارو organize میکنم داخل ابسیدین
قبلاً کیندل هم استفاده میکردم (با Readwise) ولی الان به دلایلی استفاده نمیکنم
با chatGPT هم یه فرآیند ساختم که خواندن بعضی کتابهارو برام ساده تر کرده (توضیح اینم سخته چون توضیحات زیادی داره. یه دوره براش ساختم)
از یه سری تکنیکهای حافظه و تصویرسازی هم استفاده میکنم که بتونم قطعات این پازل پیچیده رو به هم وصل کنم و توی ذهنم داشته باشم. کلاً هم تصویری فکر میکنم. مثلاً Simple Habits for Complex times توی ذهن من میشه یه سری rule که یه کاراکتر بازی توی یه محیط پیچیده طبق اونها عمل میکنه به صورت عادت اتوماتیک. این کاراکتره هم بر اساس مفاهیم پدیدارشاسانه! صدها موضوع بهش وصله. اصول بازی هم در موردش پیاده میشه (بر اساس کتاب 10,000 Hours of Play که خودم صدها ساعت بهش فکر کردم و شخصیش کردم).
فکر میکنم این بی سر و ته ترین کامنتی بود که در طول عمرم نوشته بودم. همین نشون میده که این موضوعات رو نمیتونم به این شکل خوب توضیح بدم.

**Reply by pourmand1376** [reply]
سلام مجدد هیوا جان. 

ممنون از کامنت طولانی‌ای که برام گذاشتی. یه چیزهایی دستگیرم شد ولی صادقانه بخوام بگم زیاد نفهمیدم داستان از چه قراره. واقعاً مشتاقم که یه زمانی فرصت کنی یه سری ویدئو برامون ضبط کنی، ببینم چطوریه.
